### ISSUE TITLE
[Bug] Notification Spam

### ISSUE THREAD

**Issue by FreshLlamanade** [YTVanced/Vanced#1381]
**Bug description**
As of a few hours ago, Vanced has started spamming my phone with notifications. Every 30 seconds I receive the same 5 notifications again: 3 alerting video recommendations, one comment reply and one silent video recommendation. They are always about the same videos. Clearing the notifications does not prevent them from reappearing again. I have force stopped the program and cleared notifications, but as soon as I start Vanced again the notifications start coming again. 

**Variant**
Non-Root

**Vanced version**
v16.29.39

**Android version**
Android 11

**Device**
Samsung Galaxy S20 FE
fully updated as of 12 Nov 21

**Steps to Reproduce**
Steps to reproduce the error:
1. Open the app
I wish I knew what the cause was. But it is ongoing and persistent.

**Expected behavior**
Video recommendations come once and only once.

https://user-images.githubusercontent.com/59163924/141538780-9f9005a1-8ff6-435d-b30d-70e1d4eb4471.mp4

**Comment by milindgoel15** [comment]
Duplicate of #1107
